### PR TITLE
Fill model attributes

### PR DESCRIPTION
**Pull request by poojaRal**
Please check all the attributes I filled in for the tables without joins

### CODE REVIEW

File: model/survey-instance.js [hunk 1]
```diff
@@ -21,8 +21,33 @@ const Sequelize = require('sequelize');
...
+            },
+            actualSubmissionTime: {
+                type: Sequelize.DATE
+            },
+            surveyInstanceCompleted: {
+                type: Sequelize.BOOLEAN,
+                allowNull: false,
+                defaultValue: false,
+                validate: {
+                    notEmpty: true
```
Comment (ChristianMurphy) [inline model/survey-instance.js]: will `allowNull: false` and `defaultValue: false` the boolean doesn't need to be validated `notEmpty`

File: model/patient.js [hunk 2]
```diff
@@ -23,6 +23,19 @@ module.exports = function (sequelize) {
         {
             pin: {
                 type: Sequelize.INTEGER
+            },
+            deviceType: {
+                type: Sequelize.ENUM,
+                values: ['android','ios','windows']
+            },
+            version: {
```
Comment (ChristianMurphy) [inline model/patient.js]: `version` keeps on confusing me.
`deviceVersion` may be more clear.

**Comment by ChristianMurphy** [comment]
Other than comments and lint checks.
LGTM

File: model/survey-instance.js (hunk 1 above)
Comment (poojaRal) [inline model/survey-instance.js]: I removed the validator. Will that suffice? I thought of a default value as false to be present for this boolean. Hence i should get rid of allowNull right?

File: model/patient.js (hunk 2 above)
Comment (poojaRal) [inline model/patient.js]: Changed it.

**Comment by ChristianMurphy** [comment]
:ship: it

**Pull request merged**